doc/source/computation.rst

```diff
@@ -47,17 +48,18 @@ NA/null values *before* computing the percent change).
 Covariance
 ~~~~~~~~~~
 
-The ``Series`` object has a method ``cov`` to compute covariance between series
-(excluding NA/null values).
+The ``Series`` object has a method :meth:`~Series.cov` to compute covariance 
+between series (excluding NA/null values).
 
 .. ipython:: python
 
    s1 = pd.Series(np.random.randn(1000))
    s2 = pd.Series(np.random.randn(1000))
    s1.cov(s2)
 
-Analogously, ``DataFrame`` has a method ``cov`` to compute pairwise covariances
-among the series in the DataFrame, also excluding NA/null values.
+Analogously, ``DataFrame`` has a method :meth:`~DataFrame.cov` to compute 
```

Comment:
And 

    Analogously, :meth:`DataFrame.cov` can be used to compute pairwise...